### ISSUE TITLE
JSON Mode Support

### ISSUE THREAD

**Issue by dcallahan11** [jgravelle/GroqApiLibrary#2]
Hi, does this library support JSON Mode requests to Groq? I'm attempting to replicate the response format in the JObject I'm passing to Groq, but keep receiving status code 400 responses from my calls. If I remove the "response_format" parameter below, my calls work fine, which leads me to believe that either my replication is malformed or the library doesn't have support for JSON Mode. Any guidance would be appreciated.

JObject request = new()
{
    ["model"] = modelName,
    ["temperature"] = temperature,
    ["max_tokens"] = maxTokens,
    ["top_p"] = 1,
    ["messages"] = new JArray
        {
            new JObject
            {
                ["role"] = "system",
                ["content"] = systemPrompt
            },
            new JObject
            {
                ["role"] = "user",
                ["content"] = userPrompt
            }
        },
    ["response_format"] = new JObject
    {
        ["type"] = "json_object",
    }
};

**Comment by jgravelle** [comment]
Updated for you.  Let me know how it works...!  -jjg

## Latest Update
Added the response_format parameter to the request, ensuring that the response from Groq Cloud is in JSON format.

This modification should not impact existing client applications that are using this library, as long as they do not explicitly set the response_format parameter in their requests. The library will still work correctly for them, and their client-side code will not need refactoring.
The modification only adds the response_format parameter if it is not already present in the request. This ensures that existing applications will not be affected, while also allowing JSON Mode requests to function correctly.
In summary, there should be no impact on existing client applications, and no client-side code refactoring is required.
